network/transparent_vlan_endpointclient_linux.go

```diff
@@ -171,7 +232,7 @@ func (client *TransparentVlanEndpointClient) PopulateVM(epInfo *EndpointInfo) er
 		// We assume the only possible error is that the namespace doesn't exist
 		logger.Info("No existing NS detected. Creating the vnet namespace and switching to it")
 
-		if err = client.createNetworkNamespace(vmNS, numRetries); err != nil {
+		if err = client.createNetworkNamespace(vmNS); err != nil {
```

Comment:
why dont we call this also with RunwithRetries instead of calling within the method?